ui/selectors/multichain.ts

```diff
@@ -387,7 +380,7 @@ function getNonEvmCachedBalance(state: MultichainState) {
       network.chainId as MultichainNetworks
     ]?.[0];
 
-  return balances?.[account.id]?.[asset]?.amount;
+  return balances?.[selectedAccount.id]?.[asset]?.amount ?? 0;
```

Comment:
I wonder if we can handle this in a better than just a conditional chaining. Maybe adding some guards that will console.error if an attr is falsy?